CodegenCLI/Sources/main.swift

```diff
@@ -5,10 +5,9 @@ struct CodegenCLI: ParsableCommand {
   static var configuration = CommandConfiguration(
     commandName: "apollo-ios-cli",
     abstract: "A command line utility for Apollo iOS code generation.",
-    version: "1.0.0-alpha.8",
+    version: "1.0.0-beta.1",
```

Comment:
Is there some way for us to have this automatically update when we bump the version of the main repo?

Reply:
If we could link it to apollo-ios xcconfig files maybe but they are outside the project folder so I'm not sure about whether that's possible.